### PR TITLE
Even better index page

### PR DESCRIPTION
**Pull request by coffee-cup**
- Upgrade to next 13
- Featured posts
- Condensed list

### CODE REVIEW

File: components/Categories.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import Link from "./Link"
 
 export const Categories: React.FC = () => {
   return (
-    <ul className="flex flex-wrap gap-4 md:gap-8 mt-8 mb-12">
+    <ul className="flex flex-wrap gap-4 md:gap-8 mt-8 mb-8">
```
Comment (FarazPatankar) [inline components/Categories.tsx:7]: `my-8` instead of `mt-8 mb-8`?

File: components/FeaturedPostItem.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      href={`/p/${post.properties.Slug.rich_text[0].plain_text}`}
+      className="group"
+    >
+      {featuredImage != null ? (
+        <div className="w-full aspect-[2.25/1] relative border border-black border-opacity-10 rounded-xl overflow-hidden">
+          <Image
+            src={featuredImage}
+            fill
+            priority
+            alt=""
```
Comment (FarazPatankar) [inline components/FeaturedPostItem.tsx:32]: Can we add an `alt` here? Maybe the title of the post?

File: components/FeaturedPostItem.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        <header className="font-bold text-2xl my-4 group-hover:opacity-60">
+          <NotionText text={post.properties.Page.title} noLinks />
+        </header>
+
+        <p className="text-lg text-gray-800 line-clamp-2">
+          <NotionText text={post.properties.Description.rich_text} noLinks />
+        </p>
+
+        <div className="flex items-center gap-3 mt-6">
+          <img
```
Comment (FarazPatankar) [inline components/FeaturedPostItem.tsx:52]: Any reason we don't use the NextJS Image component for this?

**Review by FarazPatankar — APPROVED** [review]

File: components/FeaturedPostItem.tsx (hunk 3 above)
Comment (coffee-cup) [reply components/FeaturedPostItem.tsx:52]: The avatars are really small and all have different sizes and sources which makes using the Next image component fairly difficult

**Pull request merged**